source/dem/pw_linear_force.cc

```diff
@@ -98,10 +98,15 @@ PWLinearForce<dim>::PWLinearForce(
 
   this->calculate_force_torque_on_boundary =
     dem_parameters.forces_torques.calculate_force_torque;
-  this->center_mass_container = dem_parameters.forces_torques.point_center_mass;
-  this->boundary_index        = boundary_index;
-  this->force_on_walls        = this->initialize();
-  this->torque_on_walls       = this->initialize();
+  this->center_mass_container =
+    dem_parameters.forces_torques.triangulation_center_mass;
+  this->triangulation_mass = dem_parameters.forces_torques.triangulation_mass;
+  this->boundary_index     = boundary_index;
+  for (unsigned int d = 0; d < dim; ++d)
+    this->gravity[d] = dem_parameters.physical_properties.g[d];
+
+  this->rotation_axis = dem_parameters.grid_motion.cylinder_rotation_axis;
+  this->inclined_plane_angle = dem_parameters.grid_motion.inclined_plane_angle;
```

Comment:
This code is duplicated in both linear and non-linear force model, it should not be necessary no?